src/simplewallet/password_container.cpp

```diff
@@ -132,6 +130,38 @@ namespace tools
     return true;
   }
 
+bool password_container::read_from_tty_temp(const char *message) {
+    /**
+     **  prompt twice for password and compare; call appropriate function to set member variable
+     **/
+    std::string pass1;
+    std::string pass2;
+    bool match=false;
+    do{
+        if (message)
+            std::cout << message <<": ";
+        while (!password_container::read_from_tty(pass1));
```

Comment:
If (!...)
  return false;

Otherwise it will break the use of simplewallet with non TTY stdin